### PR TITLE
OpenFOAM on Iceberg: note that need to explicitly load the Scotch library

### PR DESCRIPTION
**Pull request by willfurnass**
Addresses #746

### CODE REVIEW

File: iceberg/software/apps/openfoam.rst [hunk 1]
```diff
@@ -15,22 +15,24 @@ OpenFOAM is free, open source software for computational fluid dynamics (CFD).
 Usage
 -----
 
-The lastest version of OpenFoam can be activated using the module file::
+You can load a specific version of OpenFOAM using one of: ::
 
-    module load apps/gcc/5.2/openfoam
+       module load apps/gcc/5.2/openfoam/3.0.0
+       module load apps/gcc/5.2/openfoam/2.4.0
```
Comment (dmrogers75) [inline iceberg/software/apps/openfoam.rst]: There is also 4.0.0 installed (by Norbert) in /usr/local/packages6/apps/gcc/5.2/openfoam:

drwxr-sr-x 3 sa_cs1ngg cs         4096 Jul 13  2016 2.4.0
drwxrwsr-x 3     27883 app-admins 4096 Nov 19  2015 3.0.0
drwxrwsr-x 3 sa_cs1ngg app-admins 4096 Jul  9  2016 4.0.0

File: iceberg/software/apps/openfoam.rst [hunk 2]
```diff
@@ -5,7 +5,7 @@ OpenFOAM
 .. sidebar:: OpenFOAM
    
    :Version: 3.0.0
-   :Dependancies: gcc/5.2 mpi/gcc/openmpi/1.10.0 scotch/6.0.4
```
Comment (dmrogers75) [inline iceberg/software/apps/openfoam.rst]: Change Dependancies to Dependencies.

File: iceberg/software/apps/openfoam.rst [hunk 3]
```diff
@@ -5,7 +5,7 @@ OpenFOAM
...
+   :Dependancies: gcc/5.2 mpi/gcc/openmpi/1.10.0 libs/gcc/5.2/scotch/6.0.4
```
Comment (dmrogers75) [inline iceberg/software/apps/openfoam.rst]: Change Dependancies to Dependencies.

File: iceberg/software/apps/openfoam.rst [hunk 4]
```diff
@@ -15,22 +15,24 @@ OpenFOAM is free, open source software for computational fluid dynamics (CFD).
...
 
-	module load apps/gcc/5.2/openfoam/2.4.0
-	module load apps/gcc/5.2/openfoam/3.0.0
+These ``module load`` commands also load :ref:`OpenMPI <openmpi_gcc_iceberg>` 1.10.0 and :ref:`GCC <gcc_iceberg>` 5.2.
 
-This has the same effect as sourcing the openfoam bashrc file, so that should
-not be needed.
+Next, make sure you also load the :ref:`Scotch <scotch>` library: ::
+
+       module load libs/gcc/5.2/scotch/6.0.4
```
Comment (dmrogers75) [inline iceberg/software/apps/openfoam.rst]: Would it be easier from a user's perspective to add module load libs/gcc/5.2/scotch/6.0.4 to the module files for OpenFoam?

File: iceberg/software/apps/openfoam.rst [hunk 5]
```diff
@@ -15,22 +15,24 @@ OpenFOAM is free, open source software for computational fluid dynamics (CFD).
...
 ------------------
 
-OpenFoam was compiled using the
-:download:`install_openfoam.sh </iceberg/software/install_scripts/apps/gcc/5.2/openfoam/install_openfoam.sh>` script, the module
-file is
-`3.0.0 </iceberg/software/install_scripts/apps/gcc/5.2/openfoam/3.0.0>`.
+OpenFOAM was compiled using the
+:download:`install_openfoam.sh</iceberg/software/install_scripts/apps/gcc/5.2/openfoam/install_openfoam.sh>` script.
+
+The :download:`3.0.0 module file</iceberg/software/install_scripts/apps/gcc/5.2/openfoam/3.0.0>`.
```
Comment (dmrogers75) [inline iceberg/software/apps/openfoam.rst]: Include the module file for 2.4.0 (and the missing module file for 4.0.0)?

File: iceberg/software/apps/openfoam.rst [hunk 6]
```diff
@@ -5,7 +5,7 @@ OpenFOAM
 .. sidebar:: OpenFOAM
    
    :Version: 3.0.0
```
Comment (dmrogers75) [inline iceberg/software/apps/openfoam.rst]: Change Version to Versions and add 2.4.0 (and 4.0.0, see below)?

File: iceberg/software/apps/openfoam.rst (hunk 1 above)
Comment (willfurnass) [reply iceberg/software/apps/openfoam.rst]: I don't think we should document this unless we have a modulefile and install script

File: iceberg/software/apps/openfoam.rst (hunk 4 above)
Comment (willfurnass) [reply iceberg/software/apps/openfoam.rst]: Done

File: iceberg/software/apps/openfoam.rst (hunk 5 above)
Comment (willfurnass) [reply iceberg/software/apps/openfoam.rst]: Added modulefiles for 3.0.0 (which had the wrong url) and for 2.4.0

**Pull request merged**